### ISSUE TITLE
Make pass/fail match grades

### ISSUE THREAD

**Issue by bmtcril** [openedx/xapi-db-load#117]
Currently the pass/fail state of a user is randomly generated, as are the current grades. This creates confusing visuals where the number of passing students is completely divorced from their grades. We should tie those events together to make visualizations less confusing.

**Comment by saraburns1** [comment]
`course_grade` comes from any [grading events](https://github.com/openedx/aspects-dbt/blob/main/models/grading/fact_learner_course_grade.sql/#L22) where the object_id has 'course' 
`approving_state` comes from only passed/failed [events](https://github.com/openedx/aspects-dbt/blob/main/models/grading/fact_learner_course_status.sql/#L23-L25)

so even outside of load xapi data, the pass/fail status doesn't necessarily line up with the grade (ie, there could be an `earned` event without a `passed` event)

**Comment by saraburns1** [comment]
To help with this, we will use the 'earned' events grade classification in combination with the passed/failed verbs

Passed/failed verb events do NOT have a score associated with them so there is still a possibility of a learner with 'passed' status and '0' score

**Comment by bmtcril** [comment]
Yeah, part of this issue is that what constitutes a passing grade is configurable per course (and so can change over time), so we count on the LMS to just tell us when a student is passing / failing.

**Comment by bmtcril** [comment]
Just to write down what we discussed, I think that's correct as long as we're only looking at the course level events.
